src/Storages/MergeTree/MergeTreeIndexBloomFilterText.cpp

```diff
@@ -444,11 +444,11 @@ bool MergeTreeConditionBloomFilterText::traverseTreeEquals(
         if (key_function_node_function_name == "arrayElement")
         {
             /** Try to parse arrayElement for mapKeys index.
-              * It is important to ignore keys like column_map['Key'] = '' because if key does not exists in map
+              * It is important to ignore keys like column_map['Key'] = '' because if key does not exist in map
               * we return default value for arrayElement.
```

Comment:
```suggestion
              * It is important to ignore keys like column_map['Key'] = '' because if key does not exist in the map
              * we return default the value for arrayElement.
```